modules/currency/utils/GetDigitalCurrencyBalance.ts

```diff
@@ -0,0 +1,18 @@
+import addressBalance from '../consts/AddressBalance';
+import digital_currency_units from '../data/digital-currency-units';
+
+const getDigitalCurrencyBalance = (network: string, address: string) : addressBalance => {
+
```

Comment:
Can you explain what the address parameter is for I can not see it being currently used.